variables.tf

```diff
@@ -25,6 +25,11 @@ variable "client_dns_server" {
   description = "The address of the DNS server to be pushed to the VPN clients."
 }
 
+variable "client_motd_url" {
```

Comment:
Why is this a required variable?

Reply:
It is required by our security team.  You are not allowed to not have a motd.

Reply:
Right.  I guess the point is that maybe someone would want to use this TF module outside of the COOL and our security requirements.

Reply:
Yup.  Took a second for me to think about this and realize this wasn't a `cool-*` repo.  Good catch all.

Reply:
The same thing goes for the timeout, although I assumed you could set that to `-1`.

Reply:
The value would be `0`.  See inline docs:
https://github.com/cisagov/openvpn-server-tf-module/blob/2b1735823c35e93e06ab36ba6d30dbc4b85c06e5/cloudinit/openvpn-config.tpl.yml#L65-L66